runner/src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -736,7 +736,8 @@ export class PageControllerBase {
     }
     // setting the feedbackLink to undefined here for feedback forms prevents the feedback link from being shown
     if (this.def.feedback?.url) {
-      viewModel.feedbackLink = this.feedbackUrlFromRequest(request);
+      // if externalFeedback is set to true, use the url as is instead of relative url
+      viewModel.feedbackLink = this.model.def.externalFeedback ? this.def.feedback.url : this.feedbackUrlFromRequest(request);
```

Comment:
Instead of using an additional flag, could you just update `feedbackUrlFromRequest` to check if the URL is relative? i.e. 
```
    const feedbackUrl = this.model.def.feedback?.url;
    if (feedbackUrl) {
      if (feedbackUrl.startsWith("http")) {
        return feedbackUrl;
      }

      const relativeFeedbackUrl = new RelativeUrl(feedbackUrl);
      const returnInfo = new FeedbackContextInfo(
        this.model.name,
        this.pageDef.title,
        `${request.url.pathname}${request.url.search}`
      );
      relativeFeedbackUrl.setParam(
        feedbackReturnInfoKey,
        returnInfo.toString()
      );
      return relativeFeedbackUrl.toString();
    }

    return undefined;
```